src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -109,3 +109,6 @@ export const ZERO = BigNumber.from(0n);
 // Wallets
 export const WALLET_TIMER_INTERVAL_MILLIS = (SECONDS_IN_MINUTE / 2) * 1000; // 30 seconds in milliseconds
 export const WALLET_PAGINATION = 10n;
+
+// VIP
+export const VIP_CODE_REGENERATE_INTERVAL = 45;
```

Comment:
For readability I would either add a comment about the fact that those are seconds or rename to `VIP_CODE_REGENERATE_INTERVAL_IN_SECONDS`.